sections/semantics-document-metadata.include

```diff
@@ -355,15 +355,16 @@
     <dd>
       <pre class="idl" data-highlight="webidl">
         interface HTMLLinkElement : HTMLElement {
-          attribute DOMString href;
-          attribute DOMString? crossOrigin;
-          attribute DOMString rel;
-          attribute DOMString rev;
-          [SameObject, PutForwards=value]readonly attribute DOMTokenList relList;
-          attribute DOMString media;
-          attribute DOMString hreflang;
-          attribute DOMString type;
-          [SameObject, PutForwards=value] readonly attribute DOMTokenList sizes;
+          [CEReactions] attribute USVString href;
```

Comment:
I would pull it out, but we have 288 total occurrences of this already in the spec. If you want to strip the spec of them completely, we should file a separate issue for that.